packages/react-ui-core/src/DateInput/DateInput.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import DatePicker from 'react-datepicker'
+
+export default class DateInput extends Component {
+
```

Comment:
Do we want to go ahead and set up the `themed` like the new components we have been writing? Something like `@themed(['DateInput'])`

Reply:
+1

Reply:
Datepicker is an external lib. So all the css class are global . so we don't need to have setup like @themed(['DateInput'])

Reply:
You should still have it imo